src/EnergyPlus/VariableSpeedCoils.hh

```diff
@@ -135,6 +135,7 @@ namespace VariableSpeedCoils {
         int WaterInletNodeNum;             // Node Number of the Water Onlet
         int WaterOutletNodeNum;            // Node Number of the Water Outlet
         PlantLocation plantLoc;
+        bool coilOperationFlag; // Set to false when the RatedCapCoolTotal from user is less than or equal to zero but not AutoSize
```

Comment:
This is not needed now?

Reply:
Nice catch--forgot to get rid of that.  And the old unit test which sent everything red.  New commit just made to get rid of this and the old unit test.